doc/source/changelog.rst

```diff
@@ -1,22 +1,6 @@
 Changelog
 =========
 
-0.7.0- 2024/04/05
------------------
-
--  fix: change shuffle multi-table process to return the right dataframe
--  fix: return metric parameter error to user
--  feat: return error to user if data contains ninf
--  feat: improve multi-table anonymization quality (utility)
--  feat: verify compatibility with server on client init
--  feat: add dataset name in the multitable privacy metrics
--  feat: create privacy geolocation assessment feature
--  refactor: add custom methods for Datasets
--  refactor: change seed place for avatarization and metrics job
-   parameters to guarantee reproducibility
-
-.. _section-1:
-
```

Comment:
Needs rebase

Reply:
That was a fresh checkout.... Can't rebase or maybe you have to teach me to time-travel.